src/components/MovieCard/MovieCard.tsx

```diff
@@ -2,6 +2,46 @@ import React from 'react';
 
 import './MovieCard.scss';
 
-export const MovieCard: React.FC = () => (
-  <>Put the card here</>
+type Props = {
+  title: string;
+  /* eslint-disable */
+  description?: string;
+  imgUrl: string;
+  imdbUrl: string;
+  imdbId: string;
+};
```

Comment:
you can use Pick for reuse separate properties of interface
`type Props = Pick<Movie, 'title' | 'description' | 'imgUrl' | 'imdbUrl'>;`